src/components/productsList/_productsList.scss

```diff
@@ -0,0 +1,121 @@
+$media-tablet: 768px;
+$media-desktop: 1024px;
+$padding-mobile: 20px;
+$padding-desktop: 40px;
+
+.products-list {
+  display: flex;
+  flex-direction: column;
+  margin: 40px 10px 10px;
+
+  * {
+    box-sizing: border-box;
+  }
+
+  @media screen and (min-width: $media-desktop) {
+    flex-direction: row;
+  }
+}
+
+.products-list-nav {
+  $self: &;
+  background: $primary;
+  position: relative;
+  padding: $padding-mobile;
```

Comment:
Try to avoid these type of variables. Make a few spacing variables that you can use for margins, paddings etc. (e.g. `spacing-xxs`, `$spacing-xs`, `spacing-s`, `$spacing-m` etc...)

Reply:
Nice, I created these variables just to understand the boilerplate and be simple. In my projects I prefer use this approach using sass maps and functions to get theses spacings :)

Reply:
Understandable 👍